action/repo_update.go

```diff
@@ -71,23 +71,26 @@ var RepoUpdate = &cli.Command{
 			Usage:   "max time allowed per build in repository",
 			Value:   30,
 		},
-		&cli.BoolFlag{
+		&cli.StringFlag{
 			EnvVars: []string{"VELA_PRIVATE", "REPO_PRIVATE"},
 			Name:    "private",
 			Aliases: []string{"p"},
 			Usage:   "disable public access to the repository",
+			Value:   "false",
 		},
-		&cli.BoolFlag{
+		&cli.StringFlag{
 			EnvVars: []string{"VELA_TRUSTED", "REPO_TRUSTED"},
 			Name:    "trusted",
 			Aliases: []string{"tr"},
 			Usage:   "elevated permissions for builds executed for repo",
+			Value:   "false",
 		},
-		&cli.BoolFlag{
+		&cli.StringFlag{
 			EnvVars: []string{"VELA_ACTIVE", "REPO_ACTIVE"},
 			Name:    "active",
 			Aliases: []string{"a"},
 			Usage:   "current status of the repository",
+			Value:   "false",
```

Comment:
Do you think this should default to `true`?

I'm wondering if this could have unintended consequences when someone goes to update a repo via the CLI.

I know when I work with the local stack, I often run:

`vela update repo --org <org> --repo <repo> --event <event_1> --event <event_2>`

I think this change would cause the `active` field to always be `false` in those scenarios.

Thoughts there?